### PR TITLE
[MRTK Shell Parity] [Slate] Audio feedback + touch point affordance

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
Adding audio feedback and touch point visualization.
Part of #4200 UX Controls Parity with HoloLens 2 Shell

- Added the latest audio clips from the shell (touch and release)
- Added PanStarted, PanStopped events to HandInteractionPanZoom.cs (following ManipulationHandler and BoundingBox pattern)
- Assigned audio clips (MRTK_Slate_Touch, MRTK_Slate_Release)
- Refined reticle prefab material and size
- Assigned left/right touch point reticle prefab (Slate prefab updated)
- Removed directional light from the slate sample content material
- Increased the margin for the bounding box to match shell design

_Center point is for the two-handed zoom. Default is none to match shell design but developers can assign it for debugging purpose._
2019-05-21 23_49_49-Unity 2018 3 13f1 Personal - HandInteractionExamples unity - MRTK-GitHub - Unive

## Changes
- Fixes: #4412 #4513

## Capture
Singe touch point visual
MRTK_ShellParity_Slate_Touchpoint2

Two touch points
MRTK_ShellParity_Slate_Touchpoint3

Updated margin for the bounding box
SlateMargin

### CODE REVIEW

File: Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs [hunk 1]
```diff
@@ -81,10 +82,10 @@ protected class HandPanData
         [Tooltip("Each object listed must have a script that implements the IHandPanHandler interface or it will not receive events")]
         private GameObject[] panEventReceivers = null;
 
-        [Header("Geometry")]
+        [Header("Visual affordance")]
         [SerializeField]
         [Tooltip("If affordace geometry is desired to emphasize the touch points(leftPoint and rightPoint) and the center point between them (reticle), assign them here.")]
-        private GameObject reticle = null;
+        private GameObject centerPoint = null;
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs]: The [FormerlySerializedAsAttribute](https://docs.unity3d.com/ScriptReference/Serialization.FormerlySerializedAsAttribute.html) should be added here, to preserve previous asset serialization.

Comment (cre8ivepark) [reply Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs]: Added it. Thanks!

**Comment by thalbern** [comment]
I'm getting an offset for that single touch point visual if you click and drag for panning  with the hand simulation
touch_point_visual
 (looks fine for panning without clicking)

**Pull request merged**